specs/chain/hs/src/Cardano/Spec/Chain/STS/Rule/Chain.hs

```diff
@@ -147,47 +147,37 @@ instance HasTrace CHAIN where
     mTxSz <- Gen.integral (Range.constant 0 4000000)
     mPSz <- Gen.integral (Range.constant 0 4000000)
 
-    -- The size of the rolling window should be equal to k.
-    -- TODO: we need to adapt the formal specs to reflect this.
-    w <- Gen.integral (Range.linear 1 10)
-
-    -- TODO: The delegation liveness parameter is equal to `2 * k`.
-    -- We should adapt the specs to reflect this.
-    d <- pure $! SlotCount $ 2 * fromIntegral w
+    -- Chain stability.
+    k <- Gen.integral (Range.linear 1 10)
```

Comment:
Why this particular range?

Reply:
No particular reason. That does look low though, so I'm bumping this to an integer between 1 and twice the value
chosen for the Byron release (2160). But the choice remains arbitrary still.